Makefile.dep

```diff
@@ -325,3 +325,8 @@ ifneq (,$(filter cpp11-compat,$(USEMODULE)))
   USEMODULE += timex
   FEATURES_REQUIRED += cpp
 endif
+
+ifneq (,$(filter hih6130,$(USEMODULE)))
+  USEMODULE += vtimer
+  FEATURES_REQUIRED += periph_i2c
```

Comment:
Is this the correct place for adding the driver dependencies? (hih6130 uses vtimer internally)
Is `FEATURES_REQUIRED` correct to put here?